src/ucp/core/ucp_request.c

```diff
@@ -328,7 +328,7 @@ void ucp_request_init_multi_proto(ucp_request_t *req,
     if (req->flags & (UCP_REQUEST_FLAG_SEND_TAG |
                       UCP_REQUEST_FLAG_SEND_AM)) {
         req->send.msg_proto.message_id  = req->send.ep->worker->am_message_id++;
-        req->send.msg_proto.am_bw_index = 1;
+        req->send.msg_proto.am_bw_index = 0;
```

Comment:
👍  it was before my change in https://github.com/openucx/ucx/pull/4792

![image](https://user-images.githubusercontent.com/11650339/80221086-76aeea80-864d-11ea-89e5-3a84abc83f81.png)